theano/tensor/tests/test_opt.py

```diff
@@ -4037,8 +4045,8 @@ def test_local_useless_tile(self):
                 f(data)
                 
                 # Check that stacktrace is copied over
-                self.assertTrue(check_stack_trace(f, ops_to_check='all'))
-                self.assertTrue(len(f.outputs[0].variable.tag.trace)>0)
+                # FIXME: DeepCopyOp has empty trace
+                #self.assertTrue(check_stack_trace(f, ops_to_check='all'))
```

Comment:
Update the comment to say that the optimization has only removed Ops, not inserted new ones, which is why there is no stack trace to check.